pkg/kubectl/cmd/cp/cp.go

```diff
@@ -93,8 +97,15 @@ func NewCmdCp(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.C
 		Use:                   "cp <file-spec-src> <file-spec-dest>",
 		DisableFlagsInUseLine: true,
 		Short:                 i18n.T("Copy files and directories to and from containers."),
-		Long:                  "Copy files and directories to and from containers.",
-		Example:               cpExample,
+		Long: templates.LongDesc(`
+		Copy files and directories to and from containers.
+
+		Local file names can be made explicit using absolute or relative
```

Comment:
It's better to extend the examples sections, not here.

Reply:
Ok! Will fix! :smile: